.storybook/main.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
       name: '@storybook/preset-scss',
       options: {
         sassLoaderOptions: {
-          // Dart Sass performs much better than Node Sass
```

Comment:
Source: https://github.com/sass/dart-sass/blob/master/perf.md

"Dart Sass on Node.js" is usually a bit slower than sassc.

Dart Sass is nice because it doesn't require installing and/or compiling native dependencies.